cheatsheets/Gitmoji.rb

```diff
@@ -27,6 +27,10 @@
         name '👷 Adding CI build system'
       end
       entry do
+        command ':bento:'
+        name '🍱 Adding or updating assets.'
```

Comment:
Whoops, I knew this. Copy pasted of the Gitmoji homepage but forgot to delete it.